src/editor.c

```diff
@@ -1886,6 +1867,53 @@ static gchar *find_calltip(const gchar *word, GeanyFiletype *ft)
 			g_ptr_array_free(constructor_tags, TRUE);
 		}
 	}
+	return tags;
+}
+
+
+static void update_tag_for_construtor_calltip(const gchar *word, TMTag *tag, const gchar **tag_name,
+											  const gchar **scope, const gchar *constructor_method)
```

Comment:
I'd suggest moving `constructor_method` in front of `tag_name` and `scope` so the input arguments are in front of the output arguments.

Reply:
Done, thanks.